### PR TITLE
Feature/infinite list

### PR DESCRIPTION
**Pull request by philippe-cleany**

### CODE REVIEW

File: e2e/tests/permissions.js [hunk 1]
```diff
@@ -12,11 +12,11 @@ describe('Permissions', () => {
     const CreatePage = createPageFactory(
         'http://localhost:8083/#/users/create'
     )(driver);
-    const EditPage = editPageFactory('http://localhost:8083/#/users/1/edit')(driver);
+    const EditPage = editPageFactory('http://localhost:8083/#/users/1')(driver);
```
Comment (sebastien-cleany) [inline e2e/tests/permissions.js]: nope, since your change on show/edit default.

File: src/reducer/admin/resource/list/ids.js [hunk 2]
```diff
@@ -36,8 +36,17 @@ export default resource => (
         return previousState;
     }
     switch (type) {
-        case CRUD_GET_LIST_SUCCESS:
-            return addRecordIds(payload.data.map(({ id }) => id), []);
+        case CRUD_GET_LIST_SUCCESS: {
+            if (meta.infiniteList) {
+                if (requestPayload.pagination.page === 1) {
```
Comment (sebastien-cleany) [inline src/reducer/admin/resource/list/ids.js]: if (!meta.infiniteList || requestPayload.pagination.page === 1) {
  return addRecord();
}

File: src/mui/list/InfiniteList.js [hunk 3]
```diff
@@ -0,0 +1,216 @@
...
+const getQuery = createSelector(getLocationSearch, locationSearch => {
+    const query = parse(locationSearch);
+    if (query.filter && typeof query.filter === 'string') {
+        query.filter = JSON.parse(query.filter);
+    }
+    return query;
+});
+
+function mapStateToProps(state, props) {
+    const resourceState = state.admin.resources[props.resource];
```
Comment (sebastien-cleany) [inline src/mui/list/InfiniteList.js]: See my previous comment

**Review by sebastien-cleany — CHANGES_REQUESTED** [review]

File: e2e/tests/permissions.js (hunk 1 above)
Comment (philippe-cleany) [reply e2e/tests/permissions.js]: it should not be commited

File: src/mui/list/InfiniteList.js (hunk 3 above)
Comment (philippe-cleany) [reply src/mui/list/InfiniteList.js]: I trying this way but it doesn't work, I propose to add hasmorre props to the list and import it to infiniteList component

**Review by sebastien-cleany — APPROVED** [review]

**Pull request merged**